.github/workflows/pa11y.yml

```diff
@@ -1,9 +1,8 @@
 name: pa11y
 
 on:
-  pull_request:
-    branches:
-      - main
+  schedule:
+    - cron: '5 4 * * *'
```

Comment:
The requirement is "integrate pa11y with CI pipeline to run on each UI pull request", which this isn't

Reply:
Can you request a review from Jason, please :)

Reply:
made the edits for this

Reply:
Sit down and speak with Kaja, they have their pa11y system working with CI pipeline. Make sure you understand how it is implemented